test/fixtures/flickr-search-app.html.out

```diff
@@ -36,7 +36,7 @@
         this.debounce('inputChanged', this.setOutput, this.delay);
       },
       setOutput: function () {
-        this.output = this.input;
+        this.set('output', this.input);
```

Comment:
I don't think this should use `set`. Since `output` is a property, it's been replaced by a getter/setter pair and updates templates and bindings automatically.